### DISCUSSION TITLE
Porting: Phone Number Validation Wireframes

### DISCUSSION THREAD

**Discussion by bhoomasri-ux** [twilio-labs/paste#1764] [Office hours] [answered]
Background: 
Stage of design: Wireframing
[Figma (Page: Wireframes v6)](https://www.figma.com/file/zh1CvZr7vdiTj6e2qM5vjb/Number-Portability-Phase-2?node-id=170%3A56982)

**Questions for OH:**

- Any existing patterns that provide users with editing options from results tables?
- Status column in the results table
- Links in Help text - Aye or Nay?
- Existing patterns where users can copy results from a table?
- Using the disclosure component to house Filter/Sort?

**Comment by gloriliale** [comment]
Thanks for posting this @bhoomasri-ux! We'll talk about it in Office Hours then link the video of it here for reference after. See ya then!

**Comment by TheSisb** [comment, marked as answer]
Hi @bhoomasri-ux,

Circling back regarding the "Portability Status" column in the table we looked at.  I checked in with a designer on our team and they pointed out that we now have a "Status pattern" in Figma which would work.  There’s also a status variant of a table cell in the table component in Figma that uses the new status pattern, for additional guidance.

The office hours recording has been posted in the #help-design-system channel in Slack as well.

Hope this helps!
